platform/ios/src/MGLMapView+IBAdditions.h

```diff
@@ -31,6 +31,8 @@ NS_ASSUME_NONNULL_BEGIN
 @property (nonatomic) IBInspectable double latitude;
 @property (nonatomic) IBInspectable double longitude;
 @property (nonatomic) IBInspectable double zoomLevel;
+@property (nonatomic) IBInspectable double minimumZoomLevel;
+@property (nonatomic) IBInspectable double maximumZoomLevel;
```

Comment:
It isn’t necessary to redeclare these properties, since identical declarations are already present on the main MGLMapView interface. Just add the `IBInspectable` keyword to those existing declarations.